src/completers/docker.zsh

```diff
@@ -47,32 +47,34 @@ _fzf_complete_docker() {
     fi
 
     if [[ $subcommand = 'inspect' ]]; then
-        local inspect_type_idx=${${(Q)${(z)arguments}}[(i)--type]}
-
-        if [[ -z ${${(Q)${(z)arguments}}[(r)--type*]} ]] ||
-            [[ -n ${${(Q)${(z)arguments}}[(r)--type=container]} ]] ||
-            [[ ${${(Q)${(z)arguments}}[inspect_type_idx+1]} = 'container' ]]; then
-            _fzf_complete_docker-containers '--all' '--multi' $@
-            return
-        fi
-
-        if [[ -n ${${(Q)${(z)arguments}}[(r)--type=image]} ]] ||
-            [[ ${${(Q)${(z)arguments}}[inspect_type_idx+1]} = 'image' ]]; then
-            _fzf_complete_docker-images '--multi' $@
-            return
-        fi
-
-        if [[ -n ${${(Q)${(z)arguments}}[(r)--type=network]} ]] ||
-            [[ ${${(Q)${(z)arguments}}[inspect_type_idx+1]} = 'network' ]]; then
-            _fzf_complete_docker-networks '--multi' $@
-            return
-        fi
-
-        if [[ -n ${${(Q)${(z)arguments}}[(r)--type=volume]} ]] ||
-            [[ ${${(Q)${(z)arguments}}[inspect_type_idx+1]} = 'volume' ]]; then
-            _fzf_complete_docker-volumes '--multi' $@
-            return
-        fi
+        local inspect_type
+        inspect_type=($(_fzf_complete_parse_option_arguments '' '--type' $arguments || :))
```

Comment:
What happens if multiple `--type=` are passed?

Reply:
It seems that only the last `--type` be valid. The items according to the argument of the first `--type` is shown in this implementation. I will fix it.